networkx/drawing/layout.py

```diff
@@ -617,3 +619,178 @@ def flatten(l):
         else:
             yield el
 
+
+def kamada_kawai(G, pos=None, K=1.0, L=0.1, iterations=None,
+                 tolerance=1, D=0.0, weight='weight'):
+    """Position nodes using Kamada-Kawai force-directed algorithm.
```

Comment:
This means you'll need to represent this as `--`. You could also specify a file encoding of utf8 and then directly type in `–`. But if any user is not using a unicode supported terminal, then there will be a weird looking character when they view the docstring. It's 2015, and somehow, I don't think we need to continue worrying about ascii-only viewers. The only other concern is how Sphinx deals with it, and I'd guess it just works in both cases, converting -- to an en dash as appropriate.